aospy/test/test_calc_basic.py

```diff
@@ -37,6 +39,10 @@ def test_annual_mean(self):
         calc.compute()
         assert isfile(calc.path_out['av'])
         assert isfile(calc.path_tar_out)
+        data = xr.open_mfdataset(calc.path_out['av'], decode_times=False)
```

Comment:
I think `xr.open_dataset` should be all that is needed here.  I suspect `decode_times=False` is also not needed as well.